src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -380,7 +380,7 @@
             contact-request @(re-frame/subscribe [:chats/sending-contact-request])]
         [rn/view {:style     (styles/toolbar)
                   :on-layout on-chat-toolbar-layout}
-         ;;EXTENSIONS and IMAGE buttons
+         ;; EXTENSIONS and IMAGE buttons
```

Comment:
Well, I think we can ditch such comments, the code speaks for itself. There's always that fine recommendation to eliminate comments that describe the WHAT by creating separate, smaller functions with proper names.

Reply:
i guess if @OmarBasem  don't use idea to format, above comment won't appear :)